common/tools/eslint-plugin-azure-sdk/src/utils/verifiers.ts

```diff
@@ -42,6 +44,15 @@ export type VerifierMessageIds = keyof typeof VerifierMessages;
 export const stripPath = (pathOrFileName: string): string =>
   pathOrFileName.replace(/^.*[\\\/]/, "");
 
+export function usesTshy(packageJsonPath: string) : boolean {
+  const dotTshy = path.join(path.dirname(packageJsonPath), ".tshy");
```

Comment:
You could also check the `devDependencies` for `tshy` as well as the `tshy` section in the `package.json`

Reply:
I didn't do any benchmarking but was thinking maybe trying to access a directory is faster than reading the file then check for sub string.